include/IECore/Canceller.h

```diff
@@ -76,26 +76,11 @@ class IECORE_API Canceller : public boost::noncopyable
 
 	public :
 
-		Canceller()
-			:	m_cancelled( false )
-		{
-		}
+		Canceller();
 
-		void cancel()
-		{
-			m_cancelled = true;
-		}
-
-		static void check( const Canceller *canceller )
-		{
-			/// \todo Can we reduce overhead by reading
-			/// with a more relaxed memory ordering here?
-			if( canceller && canceller->m_cancelled )
-			{
-				throw Cancelled();
-			}
-		}
+		void cancel();
 
+		static void check(const Canceller *canceller);
```

Comment:
I'm concerned about this no longer being inlined. We call `check()` from various performance-critical inner loops, and I did a fair bit of testing to ensure that it didn't have too much overhead. Not inlining potentially changes the balance there - is there some other way we can address the MSVC issue?

Reply:
Looks like an easy fix of just removing the export declaration which allows it to be inlined as intended.